src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -127,7 +127,7 @@ class JesBackend extends Backend with LazyLogging {
       val evaluatedExpression = taskOutput.expression.evaluate(scopedLookupFunction, engineFunctions, interpolateStrings = true)
       evaluatedExpression match {
         case Success(v) =>
-          val jesOutput = JesOutput(taskOutput.name, s"$callGcsPath/${taskOutput.name}", Paths.get(v.valueString))
+          val jesOutput = JesOutput(taskOutput.name, s"$callGcsPath/${v.valueString}", Paths.get(v.valueString))
```

Comment:
I'm not actually sure what this change fixes or why it was made... can you explain?